packages/strapi-plugin-graphql/package.json

```diff
@@ -12,7 +12,7 @@
   },
   "dependencies": {
     "@apollo/federation": "^0.15.0",
-    "apollo-server-koa": "2.9.12",
+    "apollo-server-koa": "^2.14.2",
```

Comment:
Please make use of strict versions :)

Reply:
You mean like `2.14.2`? I'm happy to chnage but can you explain the reasoning for strict versioning on just this dependency?

Reply:
@IkeLutra it is not specific to this package but we are taking every occasion to move to strict versions :) We have a PR to add dependabot PRs so we can get automated upgrades for thise kind of issues and still have stable versions so we don't break in unintended ways when dependecies break (which happened a few times already) Hope this helps

Reply:
Thanks that is really helpful. Just wanted to understand the reasoning that makes a lot of sense. Will update now 😄